### PR TITLE
Added specific rules for ITA

### PR DESCRIPTION
**Pull request by salesfelipe**
#### What is the purpose of this pull request?
 - ditto.
 
#### What problem is this solving?
- Italian stores don't require document.
- https://vtex-dev.atlassian.net/secure/RapidBoard.jspa?rapidView=109&modal=detail&selectedIssue=POST-673
- Same as: https://github.com/vtex/profile-form/pull/91

#### How should this be manually tested?

- https://felipe--arcaplanet.myvtex.com/account?__inspect#/profile/edit

#### Types of changes
- [ ] Bug fix (a non-breaking change which fixes an issue)
- [x] New feature (a non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Requires change to documentation, which has been updated accordingly.

### CODE REVIEW

File: react/rules/ITA.js [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+      maxLength: 50,
+      label: 'document',
+      required: false,
+      hidden: true,
+    },
+    {
+      name: 'homePhone',
+      maxLength: 30,
+      label: 'homePhone',
+    },
```
Comment (estacioneto) [inline react/rules/ITA.js:35]: No phone validation?

File: react/rules/ITA.js [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+    {
+      name: 'tradeName',
+      maxLength: 100,
+      label: 'tradeName',
+    },
+    {
+      name: 'corporateDocument',
+      maxLength: 30,
+      label: 'corporateDocument',
+      required: false,
```
Comment (estacioneto) [inline react/rules/ITA.js:64]: Curiosity: Why are some of the fields explicitly `required: false` and others aren't?

**Review by estacioneto — APPROVED** [review]
Just two questions. I don't know profile-form that well to deeply understand what's going on, but the code LGTM.

File: react/rules/ITA.js (hunk 1 above)
Comment (salesfelipe) [reply react/rules/ITA.js:35]: Nops, I'm justing using the default validations and changed the one for document

File: react/rules/ITA.js (hunk 2 above)
Comment (salesfelipe) [reply react/rules/ITA.js:64]: To override the default rules

**Pull request merged**